research/query_service/ir/graph_proxy/src/gs_store/graph_query.rs

```diff
@@ -89,23 +86,50 @@ where
                 .iter()
                 .map(|pid| *pid as PartitionId)
                 .collect();
-            let result = store
-                .get_all_vertices(
-                    si,
-                    label_ids.as_ref(),
-                    // None means no filter condition pushed down to storage as not supported yet. Same as follows.
-                    None,
-                    // None means no need to dedup by properties. Same as follows.
-                    None,
-                    prop_ids.as_ref(),
-                    // Zero limit means no limit. Same as follows.
-                    params.limit.unwrap_or(0),
-                    // Each worker will scan the partitions pre-allocated in source operator. Same as follows.
-                    partitions.as_ref(),
-                )
-                .map(move |v| to_runtime_vertex(&v));
+            if let Some(indexed_values) = params.get_extra_param("PK") {
```

Comment:
Consider a index scan interface in `GraphProxy`.

Reply:
Done